src/FsAutoComplete/LspServers/AdaptiveFSharpLspServer.fs

```diff
@@ -793,9 +787,10 @@ type AdaptiveFSharpLspServer
         checker.ClearCaches() // if we got new projects assume we're gonna need to clear caches
 
         let options =
-          projectOptions
-          |> List.map (fun o ->
-            let fso = FCS.mapToFSharpProjectOptions o projectOptions
+          let fsharpOptions = projectOptions |> FCS.mapManyOptions |> Seq.toList
```

Comment:
This is the main change. We had this in `Ionide.ProjInfo` but weren't using it. This prevent recreation of referenced types.